spyder/widgets/tests/test_shortcutssummary.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+#
+
+"""
+Tests for the Shortcut Summary Widget.
+"""
+import sys
+
+import pytest
+from qtpy.QtCore import Qt
+
+from spyder.widgets.shortcutssummary import ShortcutsSummaryDialog
+from .helpers import SHORTCUTS
+
+
+@pytest.fixture
+def dlg_shortcuts(qtbot):
+    """Set up shortcut summary widget."""
+    dlg_shortcuts = ShortcutsSummaryDialog(None)
+    qtbot.addWidget(dlg_shortcuts)
+    dlg_shortcuts.show()
+    yield dlg_shortcuts
+    dlg_shortcuts.close()
+
+
+def test_shortcutssummary_exists(dlg_shortcuts, qtbot):
+    """Test that shortcut summary is visible and is not empty"""
+    # Test that the dialog exists and is shown
+    assert dlg_shortcuts.isVisible()
+
+    # Test that the dialog is not empty
+    assert not dlg_shortcuts._layout.isEmpty()
+
+    # Test that the dialog is closed properly on Esc keypress
+    qtbot.keyClick(dlg_shortcuts, Qt.Key_Escape)
+    assert not dlg_shortcuts.isVisible()
+
+
+def test_shortcutssummary_texts(dlg_shortcuts, qtbot):
+    """Test that each shortcut has platform-specific key names."""
+    expected_shortcuts = 124
```

Comment:
I think this should be 125 because the shortcut `Copy` appears two times, one for the Editor and one for the Variable Explorer.

Reply:
mmmh, I was not correct here. I don't know why `Copy` is found two times at the end sorry.